### PR TITLE
[MIG] Migration module account_group_invoice_line to v12

### PR DESCRIPTION
**Pull request by jado95**

### CODE REVIEW

File: account_group_invoice_line/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Group Invoice Lines',
+    'version': '12.0.1.1.0',
+    'category': 'Accounting & Finance',
+    'summary': 'Add option to group invoice lines per account',
+    'author': 'SYLEAM,Akretion,Dinamiche Aziendali srl, Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'website': 'http://www.syleam.fr/',
```
Comment (nikul-serpentcs) [inline account_group_invoice_line/__manifest__.py]: https://github.com/OCA/account-invoicing

File: account_group_invoice_line/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# © 2012-2015 SYLEAM Info Services (<http://www.syleam.fr/>)
+# © 2015 Akretion (http://www.akretion.com)
+# @author: Gianmarco Conte <gconte@dinamicheaziendali.it>
+# @author: Sébastien LANGE <sebastien.lange@syleam.fr>
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Group Invoice Lines',
+    'version': '12.0.1.1.0',
```
Comment (nikul-serpentcs) [inline account_group_invoice_line/__manifest__.py]: 12.0.1.0.0

File: account_group_invoice_line/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```
Comment (nikul-serpentcs) [inline account_group_invoice_line/models/__init__.py]: @jado95 No need, Remove, please
Everywhere

File: account_group_invoice_line/models/account_invoice.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    @api.model
+    def line_get_convert(self, line, part):
+        res = super(AccountInvoice, self).line_get_convert(line, part)
+        if line.get('invoice_id'):
+            inv = self.browse(line['invoice_id'])
+            jrl = inv.journal_id
```
Comment (nikul-serpentcs) [inline account_group_invoice_line/models/account_invoice.py]: use `jrl = self.browse(line['invoice_id']).journal_id`

and remove `inv = self.browse(line['invoice_id'])`

File: account_group_invoice_line/models/account_invoice.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+            if jrl.group_invoice_lines and jrl.group_method == 'account':
+                res['name'] = '/'
```
Comment (nikul-serpentcs) [inline account_group_invoice_line/models/account_invoice.py]: use one liner like this res.update({'name': '/', 'product_id': False})

**Review by nikul-serpentcs — COMMENTED** [review]
minor change

**Comment by jado95** [comment]
@nikul-serpentcs it's ok now?

**Comment by nikul-serpentcs** [comment]
> @nikul-serpentcs it's ok now?

@jado95 Yes, Please Attending preserve the history

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

**Comment by igallart** [comment]
This module work also in V11, I have it installed taken from this pull request, but to v11.

**Comment by kittiu** [comment]
@jado95 do you still working on this?

Note: there is also another PR on the same thing. https://github.com/OCA/account-invoicing/pull/660

**Pull request closed without merging**